### ISSUE TITLE
Easiest way to create an order with phemex with a take profit percentage and how to check current P/L of order?

### ISSUE THREAD

**Issue by adamrg73** [ccxt/ccxt#8086]
What is the easiest way/sample code to create an order with phemex with a take profit percentage and also how to check current P/L of an existing active order?

Thanks

**Comment by frosty00** [comment]
I'm working on unifying the positions API, currently it isn't possible to do this with ccxt

**Comment by kroitor** [comment]
@adamrg73 we will post some examples on how to do that with implicit methods.

**Comment by adamrg73** [comment]
I have to check again, but quickly I think one of the fetch orders shows the PL for active orders/current positions correct?
For now instead of using take profit for an order, i can just execute the buy or sell based on the current PL.
But it would be great to know how to do the take profit param in the create order, I looked at the source I think I only found stoploss. Dont know if its implemented yet. let me know thanks again
